src/mesh/Data.hpp

```diff
@@ -23,6 +24,9 @@ namespace mesh {
  */
 class Data {
 public:
+  static const int EXTRAPOLATION_ORDER; // @todo currently hard-coded; we don't care about extrapolation here.
+  static const int INTERPOLATION_ORDER; // @todo currently hard-coded; should be configurable.
```

Comment:
`static` brings many problems, see https://github.com/precice/precice/issues/385
Let's be careful here.

Reply:
This is work-in-progress and will definitely vanish when we have a proper configuration.